tests/dummy/app/controllers/index.js

```diff
@@ -31,10 +38,19 @@ export default Controller.extend({
     {
       label: 'Country',
       valuePath: 'country'
+    },
+    {
+      cellComponent: 'custom-actions'
```

Comment:
Add the option `hideable: false` so that this column does not appear on the column chooser menu. Also, change this in the example snippet.